worker/api.js

```diff
@@ -0,0 +1,40 @@
+export async function handleAPIRequest(request) {
+  const url = new URL(request.url);
+  const accept = request.headers.get("accept");
+  const isJSON = accept && accept.indexOf("application/json") >= 0;
```

Comment:
I think it would be better to be more liberal here: `application/*` as well as `*/*` should be acceptable here.

Also, is it possible to use `str.includes(val)` rather than `str.indexOf(val) >= 0` in a worker?

Reply:
So you're saying we shouldn't check the accept header at all - I think that makes sense as we only ever respond with JSON anyway.

> Also, is it possible to use str.includes(val) rather than str.indexOf(val) >= 0 in a worker?

Yes, I copied this code from [here](https://github.com/denoland/deno_website2/pull/180/files#diff-ebde6d112f6a6eee03c43a03067292b6L15), but I'll change it.